crates/orderbook/openapi.yml

```diff
@@ -668,14 +668,25 @@ components:
                 be set to the `keccak256` hash of the UTF-8 encoded bytes of this string. You must
                 ensure that the signature that is part of this request indeed signed a smart contract
                 order with the `appData` field set appropriately. If this isn't the case, signature
-                verification will fail. For easier debugging it is recommended to additionally set the 
+                verification will fail. For easier debugging it is recommended to additionally set the
                 `appDataHash` field.
 
                 The field must be the encoding of a valid JSON object. The JSON object can contain
                 arbitrary application specific data (JSON key values). The optional key `backend` is
                 special. It **MUST** conform to the schema documented in `BackendAppData`.
 
+                The intended use of the other keys of the object is follow the standardized format
+                defined [here](https://github.com/cowprotocol/app-data). Example:
+
+                ```json
+                {
+                  "version": "0.7.0",
+                  "appCode": "YOUR_APP_CODE",
+                  "metadata": {}
+                }
+
                 The total byte size of this field's UTF-8 encoded bytes is limited to 1000.
+                ```
```

Comment:
This is including the comment into the JSON code block AFAICT.
Should be moved to line `687` instead, no?

Reply:
Thanks. Fixed.

Reply:
Thanks. Fixed.